src/__tests__/CompareResults/Retrigger.test.tsx

```diff
@@ -0,0 +1,181 @@
+      {
+        '1234567': { id: 'TeSt0FIBQyuzPDktQnTest', run: '0' },
+      },
+    );
+    (window.fetch as FetchMockSandbox).get(
+      'begin:https://firefox-ci-tc.services.mozilla.com/api/queue/v1/task/TeSt0FIBQyuzPDktQnTest/artifacts/public%2Factions.json',
+      {
+        actions: [
+          { hookPayload: {}, kind: 'hook', name: 'retrigger-multiple' },
+        ],
+        variables: {},
+      },
+    );
+    (window.fetch as FetchMockSandbox).get(
+      'begin:https://firefoxci.taskcluster-artifacts.net/TeSt0FIBQyuzPDktQnTest/0/public/actions.json',
+      {
+        actions: [
+          { hookPayload: {}, kind: 'hook', name: 'retrigger-multiple' },
+        ],
+        variables: {},
+      },
+    );
+    // fetch requests for new revision info
+    (window.fetch as FetchMockSandbox).get(
+      'begin:https://treeherder.mozilla.org/api/project/mozilla-central/jobs/381452501/',
+      {
+        push_id: 7891011,
+      },
+    );
+    (window.fetch as FetchMockSandbox).get(
+      'begin:https://treeherder.mozilla.org/api/project/mozilla-central/push/decisiontask/?push_ids=7891011',
+      {
+        '7891011': { id: 'TeStSeCoNDuzPDktwoTest', run: '0' },
+      },
+    );
+    (window.fetch as FetchMockSandbox).get(
+      'https://firefox-ci-tc.services.mozilla.com/api/queue/v1/task/TeStSeCoNDuzPDktwoTest/artifacts/public%2Factions.json',
+      {
+        actions: [
+          { hookPayload: {}, kind: 'hook', name: 'retrigger-multiple' },
+        ],
+        variables: {},
+      },
+    );
+    (window.fetch as FetchMockSandbox).get(
+      'begin:https://firefoxci.taskcluster-artifacts.net/TeStSeCoNDuzPDktwoTest/0/public/actions.json',
+      {
+        actions: [
+          { hookPayload: {}, kind: 'hook', name: 'retrigger-multiple' },
+        ],
+        variables: {},
+      },
+    );
+
+    render(<RetriggerButton result={result} />);
+
+    const openModalButton = await screen.findByTitle('retrigger jobs');
+
+    const user = userEvent.setup({ delay: null });
+    await user.click(openModalButton);
+
+    // select option 1 for base
+    const baseSelect = within(
+      await screen.findByTestId('base-retrigger-select'),
+    );
+
+    let selectButton = await baseSelect.findByRole('button');
+
+    fireEvent.mouseDown(selectButton);
+
+    let listbox = within(await screen.findByRole('listbox'));
+    fireEvent.click(await listbox.findByText('1'));
+
+    // select option 1 for new
+    const newSelect = within(await screen.findByTestId('new-retrigger-select'));
+
+    selectButton = await newSelect.findByRole('button');
+
+    fireEvent.mouseDown(selectButton);
+
+    listbox = within(await screen.findByRole('listbox'));
+    fireEvent.click(await listbox.findByText('1'));
+
+    const triggerJobsButton = await screen.findByText('Retrigger');
+
+    await user.click(triggerJobsButton);
+
+    expect('something').toBe(null);
```

Comment:
You can expect that the retrigger call has been made.

I think you can inspect all fetch calls with:
`console.log(window.fetch.mock.calls)`
So that you can then do a:
```
expect(window.fetch).toHaveBeenCalledWith(...)
```